chart/values.yaml

```diff
@@ -32,6 +32,16 @@ csv:
   - istio
   - servicemesh
   - envoy
+  olmFeatureDisconnected: true
+  olmFeatureFIPSCompliant: true
+  olmFeatureProxyAware: true
+  olmFeatureTLSProfiles: false
+  olmFeatureTokenAuthAWS: false
+  olmFeatureTokenAuthAzure: false
+  olmFeatureTokenAuthGCP: false
+  olmFeatureCNF: false
+  olmFeatureCNI: true
+  olmFeatureCSI: false
```

Comment:
We should probably group these under a common parent (e.g. `olmFeatures`) and remove the `olmFeature` prefix.